src/V8/Utils.cpp

```diff
@@ -81,7 +81,7 @@ fillV8Array(v8::Isolate * isolate, const DataTypePtr & arg_type, const MutableCo
         case TypeIndex::UInt8: {
             if (arg_type->getName() == "bool")
             {
-                for (int i = 0; i < size; i++)
+                for (uint64_t i = 0; i < size; i++)
```

Comment:
`size` is uint64_t type. plz check this file L68 
https://github.com/timeplus-io/proton/pull/123/files#diff-8ea7952977155059a02d62a2edd036d580926da68393a08197e2ea0ee96d8dddR68